### ISSUE TITLE
History of recent participation in athletics

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10706]
**Preferred term label:**
History of recent participation in athletics

**Synonyms**

**Definition (free text, please give PubMed ID)**
Football, basketball, soccer, skiing, etc

This sounds like something very common, but the "recent" part is very important.  This finding is the core finding of sports medicine, entered by an orthopedist.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Interphalangeal joint sprain
 Proximal interphalangeal (PIP) joint dislocation
 Distal interphalangeal (DIP) joint dislocation

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001128
- label: History of recent participation in athletics
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: Medical history of participation in sports (football, basketball, soccer, skiing) in the hours, days, or weeks prior to the current presenting complaint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10706
- synonyms:
